checkov/arm/graph_builder/graph_components/blocks.py

```diff
@@ -17,3 +17,7 @@ def __init__(
         id: str = "",
     ) -> None:
         super().__init__(name, config, path, block_type, attributes, id, GraphSource.ARM)
+
+    # for ARM we want to get the inner_attributes also from list[str]
```

Comment:
I think this comment is not 100% clear, without talking to you yesterday I wouldn't understand it.
Could you write something more specific and mention you intentionally don't use `super`?
Also - the doc for a function should be inside the function not above it 🙌